AWSClientRuntime/Sources/Regions/DefaultRegionResolver.swift

```diff
@@ -11,8 +11,12 @@ public struct DefaultRegionResolver: RegionResolver {
     public let providers: [RegionProvider]
     let logger: SwiftLogger
 
-    public init(providers: [RegionProvider] = [BundleRegionProvider(), EnvironmentRegionProvider(), ProfileRegionProvider(), IMDSRegionProvider()]) {
-        self.providers = providers
+    public init(providers: [RegionProvider]? = nil) {
+        #if os(iOS) || os(watchOS) || os(tvOS)
+        self.providers = providers ?? [BundleRegionProvider(), EnvironmentRegionProvider(), ProfileRegionProvider()]
```

Comment:
Don't think profile provider should be in here either? Don't think these devices will have anything saved at \.aws 

Thoughts?

Reply:
ah, yeah that makes sense